src/site/layouts/health_care_region_page.drupal.liquid

```diff
@@ -246,8 +246,7 @@ Example data:
                     class="vads-u-height--full vads-u-width--full">
                     <i
                       class="fa fa-deaf vads-facility-hub-cta-circle">&nbsp;</i>
-                    <span class="vads-facility-hub-cta-label">Order hearing aid
-                      batteries and prosthetic socks<i
+                    <span class="vads-facility-hub-cta-label">Order hearing aid batteries <br/> and accessories<i
```

Comment:
Can you please check with someone about the weird line breaks in this file?

Would be nice to clean those up.

Reply:
I just cleaned it up